solc/CommandLineInterface.cpp

```diff
@@ -692,7 +692,10 @@ void CommandLineInterface::compile()
 {
 	solAssert(CompilerInputModes.count(m_options.input.mode) == 1);
 
-	m_compiler = make_unique<CompilerStack>(m_fileReader.reader());
+	if (m_options.modelChecker.settings.solvers.eld)
+		m_compiler = make_unique<CompilerStack>(m_universalCallback.callback());
+	else
+		m_compiler = make_unique<CompilerStack>(m_fileReader.reader());
```

Comment:
If we're using the universal callback for StandardJSON I guess we could just as well do it for CLI to simplify things? I.e. SMT checker won't use the callback anyway if eld is not enabled anyway, right?

Reply:
Yep, wanted to do that after you checked the current version